### DISCUSSION TITLE
"Unknown" showing for prog2_time (sometimes other slots as well)

### DISCUSSION THREAD

**Discussion by Aatlab** [kellerza/sunsynk#240] [Q&A]
After an upgrade to the plugin a while back, I cannot get a value from the "prog2_time" attribute. It always shows "unknown". MQQT shows the correct value, but HA doesnt.

Not a programmer, and I have searched for an answer with no joy. Any tips to check on getting this value to show? For a while, I had the same for prog4_time, but it just came right by itself.

**Comment by kellerza** [comment]
Maybe you simply have the wrong entity added to your lovelace dashboard?
No other ideas, you could always open an issue with proper details.
